include/internal/libspdm_common_lib.h

```diff
@@ -422,6 +444,9 @@ void libspdm_internal_dump_data(const uint8_t *data, size_t size);
  **/
 void libspdm_internal_dump_hex(const uint8_t *data, size_t size);
 
+
+void libspdm_trace_message(void* message, size_t message_size);
```

Comment:
anyone using it?

Reply:
I was using it in a local build. Did not plan on checking this in at this time and did not mean to include it in the pull request.